### ISSUE TITLE
Android natives missing from JARs at MavenCentral for v3.4.x 

### ISSUE THREAD

**Issue by Ali-RS** [jMonkeyEngine/jmonkeyengine#1496]
The  jme3-android-native jars for v3.4.x that are uploaded to the Maven central repo are missing the native libraries.

Forum link: https://hub.jmonkeyengine.org/t/java-lang-unsatisfiedlinkerror-dalvik-system-pathclassloader-couldnt-find-libdecodejme-so/44458

**Comment by stephengold** [comment]
I confirmed this issue in the jme3-android-native-3.4.0-alpha3 and jme3-android-native-3.4.0-alpha4 JARs at repo1.maven.org . The jme3-android-native-3.4.0-alpha3 JAR at bintray.com does not suffer from this issue. (v3.4.0-alpha4 was intentionally not released to bintray.com .)

The output from the 3.4.0-alpha4 release job can be found at [https://github.com/jMonkeyEngine/jmonkeyengine/runs/2168571812](https://github.com/jMonkeyEngine/jmonkeyengine/runs/2168571812) .

**Comment by stephengold** [comment]
I think this issue is caused by specifying "-PskipPrebuildLibraries=true" in line 332 of main.yml file. I plan to remove this flag and test by releasing another alpha.

**Comment by stephengold** [comment]
I tried releasing v3.4.0-alpha5 with the fix. In the staging repository at Sonatype, I noticed that the Android natives JAR was empty (length = 352 bytes instead of 3.7 MBytes) so apparently the fix failed. A possible clue in the [job output](https://github.com/jMonkeyEngine/jmonkeyengine/runs/2249166144) at lines 194-195:
```text
> Task :jme3-android-native:copyPreCompiledLibs NO-SOURCE
> Task :jme3-android-native:copyPreCompiledOpenAlSoftLibs NO-SOURCE
```
In local builds (that generate a non-empty JAR) those tasks perform some actual work:
```text
> Task :jme3-android-native:copyPreCompiledLibs
> Task :jme3-android-native:copyPreCompiledOpenAlSoftLibs
```

**Comment by stephengold** [comment]
The 2nd try looks better in this regard and will be released as v3.4.0-alpha6

**Comment by stephengold** [comment]
I assume this issue is fixed in v3.4.0-alpha6, but I'd appreciate confirmation from an Android developer.

**Comment by Ali-RS** [comment]
Works fine. 

Thank you so much @stephengold.
